hs_core/discovery_parser.py

```diff
@@ -78,9 +78,8 @@ class ParseSQ(object):
     OP = {
         'AND': operator.and_,
         'OR': operator.or_,
-        'NOT': operator.inv,
-        '+': operator.and_,
-        '-': operator.inv,
+        'NOT': operator.inv,  # alias '-' removed 4/5/2019
+        # alias '+' (for include) removed 4/5/2019
```

Comment:
I'm not a fan of commenting out code to remove it.  The git commit history documents this for us without cluttering our code.  I am goo with putting in a comment that explains why + and - were removed intentionally.